enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -121,7 +123,7 @@ func (c *podmanCommandContainer) Create(ctx context.Context, workDir string) err
 }
 
 func (c *podmanCommandContainer) Exec(ctx context.Context, cmd *repb.Command, stdin io.Reader, stdout io.Writer) *interfaces.CommandResult {
-	podmanRunArgs := []string{}
+	podmanRunArgs := make([]string, 0, len(cmd.GetEnvironmentVariables())+len(cmd.Arguments)+1)
```

Comment:
ubernit: `2 * len(cmd.GetEnvironmentVariables())` here, or use the combined form like `fmt.Sprintf("--env=%s=%s")`

Reply:
Done